guide/intro/chapter-1/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: What is Pygame
+next_page: [/guide/intro/chapter-2, Opening A Window]
+---
+
+<body>
+  <p>In this article, we'll go over what pygame is and how to install it.</p>
+
+  <h2>Table of contents</h2>
+  <p></p>
+  <ol id="toc">
+    <!--
+            To be filled by javascript
+            -->
+  </ol>
+
+  <h2><a id="introduction" class="js">Introduction</a></h2>
+  <p>
+    It's time to get started on your pygame journey, but what is pygame? Well
+    pygame is "a free and open-source cross-platform library for the development
+    of multimedia applications like video games using Python". As the
+    description suggests, it's mostly used to make video games - which is why in
+    this book we'll be learning techniques relevant to making games
+  </p>
+
+  <p>
+    Pygame provides the very basic utility to draw and manipulate pixels on to
+    the screen, much like <a href="https://www.libsdl.org/">SDL2</a>, but with
+    extended features in various forms that makes it easier to use. Given this
+    nature, you generally have a lot of control on how you want your code to
+    run, so it may have a bit of a learning curve. But fear not! After reading
+    this book you should be able to use pygame just fine, as long as you get
+    your practice in 🔫
+  </p>
+
+  <p>
+    In this book we'll aim to cover how to do basic things with pygame, with
+    little exercises in between to make sure you're not falling off, goodluck
```

Comment:
the `goodluck` here seems wrong, here's an approximation of what I would prefer to see

```suggestion
    little exercises in between to make sure you're not falling off
    
    <b>Good Luck!</b>
```